File: C#/Car.cs
using System;

class Car : Vehicle
{
    private float fuelEfficiency;

    public Car(string vehicleId, string model, float fuelLevel, float fuelEfficiency) 
        : base(vehicleId, model, fuelLevel)
    {
        this.fuelEfficiency = fuelEfficiency;
    }

    public float GetFuelEfficiency()
    {
        return fuelEfficiency;
    }

    public void SetFuelEfficiency(float fuelEfficiency)
    {
        this.fuelEfficiency = fuelEfficiency;
    }

    public override string CalculateRange()
    {
        return (fuelEfficiency * FuelLevel).ToString();
    }
}

File: Java/Car.java
public class Car extends Vehicle {
    private double fuelEfficiency;

    public Car(String vehicleId, String model, double fuelLevel, double fuelEfficiency) {
        super(vehicleId, model, fuelLevel);
        this.fuelEfficiency = fuelEfficiency;
    }

    public double getFuelEfficiency() {
        return fuelEfficiency;
    }

    public void setFuelEfficiency(double fuelEfficiency) {
        this.fuelEfficiency = fuelEfficiency;
    }

    public double calculateRange() {
        return fuelEfficiency * getFuelLevel();
    }
}